src/main/java/com/razorpay/Utils.java

```diff
@@ -22,25 +22,26 @@ public boolean verifyPaymentSignature(JSONObject attributes) throws RazorpayExce
     String orderId = attributes.getString("razorpay_order_id");
     String paymentId = attributes.getString("razorpay_payment_id");
     String payload = orderId + '|' + paymentId;
-    return verifySignature(payload, expectedSignature);
+    String secret = this.secret.getBytes("UTF-8")
```

Comment:
@ankit-rzp No instructions for how to do that in the README. Could you help me out?